packages/design-system-tailwind-preset/scripts/testUtils.ts

```diff
@@ -0,0 +1,78 @@
+import fs from 'fs';
```

Comment:
I'm intentionally being generous with comments for these `testUtils` as they will help contributors better understand their functionality and how they can be modified when necessary.